postgres/datadog_checks/postgres/metadata.py

```diff
@@ -375,19 +377,23 @@ def _collect_postgres_schemas(self):
                 "dbms_version": self._payload_pg_version(),
                 "tags": self._tags_no_db,
                 "cloud_metadata": self._check.cloud_metadata,
+                # We don't rely on this time being strictly monotonic, it's just a unique identifier
+                # but having it be the time is helpful for debugging
+                "collection_started_at": math.floor(time.time() * 1000),
```

Comment:
Nit, `math.floor()` doesn't really buy us anything here. Since `time.time()` will always be positive number, replacing with `int()` would be the same

We could also use `time.time_ns()` if we just wanted an int value instead of a float
```python
>>> print(f"{time.time()}\n{time.time_ns()}")
1759335768.6873221
1759335768687335000
```